<ui version="4.0" >
 <class>GeneratorConfiguration</class>
 <widget class="QWidget" name="GeneratorConfiguration" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>129</width>
    <height>592</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font" >
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Signal Configuration</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="enabled_" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font" >
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text" >
        <string>On</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_6" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="leftMargin" >
       <number>4</number>
      </property>
      <property name="topMargin" >
       <number>4</number>
      </property>
      <property name="rightMargin" >
       <number>4</number>
      </property>
      <property name="bottomMargin" >
       <number>4</number>
      </property>
      <item>
       <widget class="QDial" name="frequency_" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>999</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
        <property name="sliderPosition" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="wrapping" >
         <bool>false</bool>
        </property>
        <property name="notchTarget" >
         <double>50.000000000000000</double>
        </property>
        <property name="notchesVisible" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text" >
           <string>Freq:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>1</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="frequencyValue_" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text" >
           <string>1.0</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="frequencyScale_" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font" >
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="currentIndex" >
         <number>1</number>
        </property>
        <item>
         <property name="text" >
          <string>Hz</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>kHz</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>MHz</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>GHz</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="leftMargin" >
       <number>4</number>
      </property>
      <property name="topMargin" >
       <number>4</number>
      </property>
      <property name="rightMargin" >
       <number>4</number>
      </property>
      <property name="bottomMargin" >
       <number>4</number>
      </property>
      <item>
       <widget class="QDial" name="amplitude_" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>100</number>
        </property>
        <property name="sliderPosition" >
         <number>100</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="wrapping" >
         <bool>false</bool>
        </property>
        <property name="notchTarget" >
         <double>5.000000000000000</double>
        </property>
        <property name="notchesVisible" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text" >
           <string>Amp:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>1</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="amplitudeValue_" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text" >
           <string>1.0</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="leftMargin" >
       <number>4</number>
      </property>
      <property name="topMargin" >
       <number>4</number>
      </property>
      <property name="rightMargin" >
       <number>4</number>
      </property>
      <property name="bottomMargin" >
       <number>4</number>
      </property>
      <item>
       <widget class="QDial" name="phaseOffset_" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
          <horstretch>1</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum" >
         <number>-50</number>
        </property>
        <property name="maximum" >
         <number>50</number>
        </property>
        <property name="value" >
         <number>0</number>
        </property>
        <property name="sliderPosition" >
         <number>0</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="wrapping" >
         <bool>false</bool>
        </property>
        <property name="notchTarget" >
         <double>5.000000000000000</double>
        </property>
        <property name="notchesVisible" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text" >
           <string>Phase:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>1</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="phaseOffsetValue_" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font" >
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text" >
           <string>1.0</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4" >
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QDoubleSpinBox" name="dcOffset_" >
          <property name="keyboardTracking" >
           <bool>false</bool>
          </property>
        <property name="alignment" >
         <set>Qt::AlignRight</set>
        </property>
        <property name="minimum" >
         <double>-10.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep" >
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6" >
        <property name="font" >
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>Offset (DC)</string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5" >
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="leftMargin" >
       <number>4</number>
      </property>
      <property name="topMargin" >
       <number>4</number>
      </property>
      <property name="rightMargin" >
       <number>4</number>
      </property>
      <property name="bottomMargin" >
       <number>4</number>
      </property>
      <item>
       <widget class="QComboBox" name="complexValue_" >
        <item>
         <property name="text" >
          <string>I+Q</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>I</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Q</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
